.github/workflows/push-images.yml

```diff
@@ -20,6 +20,9 @@ jobs:
 
       - uses: ./.github/actions/setup-python
 
+      - name: Install gcc
+        run: sudo apt-get update && sudo apt-get install -y gcc
```

Comment:
Does this solve the issue? I think we need to install gcc during image build.

Reply:
It really wouldn't. I've update the docker/Dockerfile to perform an update, install gcc, then clean the update directory to reduce the image size.